README.md

```diff
@@ -1,50 +1,61 @@
 | **angular-app** | [Sample Angular app](https://github.com/johnpapa/shopathome/blob/master/angular-app) | [angular.shopathome.dev](https://angular.shopathome.dev) |
-| **api**         | [Sample Azure Functions app](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
+| **api**         | [Fastify API](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
 | **react-app**   | [Sample React app](https://github.com/johnpapa/shopathome/blob/master/react-app)     | [react.shopathome.dev](https://react.shopathome.dev)     |
 | **svelte-app**  | [Sample Svelte app](https://github.com/johnpapa/shopathome/blob/master/svelte-app)   | [svelte.shopathome.dev](https://svelte.shopathome.dev)   |
 | **vue-app**     | [Sample Vue app](https://github.com/johnpapa/shopathome/blob/master/vue-app)         | [vue.shopathome.dev](https://vue.shopathome.dev)         |
 
-## Prerequisites
+## Deploy to Azure using App Spaces
 
-- A GitHub account
-- [Node.js and Git](https://nodejs.org/)
-- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=academic-0000-jopapa) installed
-- The [Static Web Apps extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps&WT.mc_id=shopathome-github-jopapa&WT.mc_id=academic-0000-jopapa) installed
-- The [SWA CLI](https://www.npmjs.com/package/@azure/static-web-apps-cli) installed
-- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=academic-0000-jopapa) installed
+[![Deploy to Azure button](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#view/Microsoft_Azure_PaasServerless/StarshotHome.ReactView)
+
+1. Once you are signed in on Azure through the link above,
+    - *For New Users:* under **Create new App Space**, you'll access the template gallery and choose a shopping app solution template based on your preferred framework.
+    - *For Existing Users:* on the page displaying your existing app spaces, Click **Create App Space** to access the template gallery and choose a shopping app solution template based on your preferred framework.
+1. Fill in the required information:
+
+    - *GitHub account* – Connect to your GitHub Account.
+    - *Organization* – Choose your organization.
+    - *New Repository* – This will be automatically populated with a fork that's created on your account.
+    - *App Space name* – Give your App Space a name (or use the default).
+    - *Subscription* – Choose your subscription.
+    - *Region* – Choose an Azure region.
+  
+1. Click **Deploy**
+
+Once your App Space is provisioned successfully, open to view your frontend deployed on Azure Static Web Apps with a link to **Open app in browser**.
+
+> [!NOTE]

```

Comment:
Even this would be done as part of the solution template, you do not need to explicitly add connection. You will end up with all components and also the components connected

Reply:
Yap! Removed that as well. Thank you